### DISCUSSION TITLE
What I missed first time opening Phoenix Slides

### DISCUSSION THREAD

**Discussion by Rigido71** [gobbledegook/creevey#56] [Ideas]
Hi,
as many users I was searching for something "different" and faster from Apple Preview and that could replace Xee.
I found a discussion on Reddit and read features, so I give it a shot.
Installed, right-click on a jpg and "Open with...PS"...everything looks good, some commands look familiar with FastStone on Windows...Great, I found the replacement!

BUT, there is something I miss:
1. If started with a file from Finder: start maximized/windowed (as user set in options), move between files is good, but it should exit pressing ESC, bypassing the Main UI.
2. I'm using a 4K 32" display with a scaled resolution of 2560x1440, maybe I'm getting older, but UI font is too small and would be great if I could choose a bigger font or making it more readable.

I really appreciated how the Main UI appears, with Column mode on top and thumbnails on bottom...nothing to say about the icon 🤣

**Comment by gobbledegook** [comment]
Esc already exits the slideshow. Cmd-Q will quit the app, if that's what you want to do!

**Reply by Rigido71** [reply]
Hi,
I said "if Phoenix is started by double-clicking an associated file from Finder, then pressing ESC should close Phoenix and bring me back to Finder". You have options for that:
<img width="457" alt="image" src="https://github.com/gobbledegook/creevey/assets/29772377/e781c1fa-bb39-41ed-959d-0980c004da6b">
You could add something like "ESC exits to Finder"

Today I also noticed that it is clearly visible Phoenix building the Main UI before going full screen and show the image I double clicked on (maybe because there are some network volumes and PS doesn't have full access to disks). Would be nicer (and user perceives as faster) if it goes immediately full screen and shows the image and, if the Main windows MUST be built, doing that off screen.

How does the association work? All file types are ticked but nothing changed on Finder. I tried to untick and retick but it didn't work. I had to do it from Finder itself...

Thank you for this great program!

**Reply by gobbledegook** [reply]
The preferences just tell Phoenix Slides what file types should be displayed. The default application when you double click on an icon in the Finder needs to be set (per file type) in the Finder.

**Reply by NuClear235** [reply]
Isnt possible to add using ENTER to quit slideshow and move back to folder, please?
(and ENTER again will return to slideshow again)

**Reply by Rigido71** [reply]
> Isnt possible to add using ENTER to quit slideshow and move back to folder, please? (and ENTER again will return to slideshow again)

Maybe, as I asked, would be good if ESC ESCapes from the program and goes back to Finder and ENTER exits from the current image and ENTERs the main GUI (the current behavior with the esc key).
